### PR TITLE
some Fixits translates to pt

### PR DESCRIPTION
**Pull request by 4SS1S**
Some translates to PT.

### CODE REVIEW

File: packages/playground-examples/copy/pt/3-7/Fixits/Big number literals.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+//// { compiler: {  target: 99 }, order: 1 }
+
+// Você sabia que existe um limite de tão grande pode
+// ser um número que você pode representar enquanto
+// você está escrevendo em Javascript?
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/3-7/Fixits/Big number literals.ts]:
```suggestion
// Você sabia que existe um limite para o maior
// número que você pode representar enquanto
// você está escrevendo em Javascript?
```

File: packages/playground-examples/copy/pt/3-7/Fixits/Const to let.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+//// { compiler: {  }, order: 1 }
+
+// No novo 3.7 a capacidade de converter rapidamente
+// uma variável const para uma let quando o valor
+// for redesignado.
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/3-7/Fixits/Const to let.ts]:
```suggestion
// No 3.7, há a nova capacidade de converter rapidamente
// uma variável const para uma let quando o valor
// for reatribuído.
```

File: packages/playground-examples/copy/pt/3-7/Fixits/Const to let.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+// Você pode tentar isso destacando o erro abaixo
+// e optando por executar a correção rápida.
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/3-7/Fixits/Const to let.ts]:
```suggestion
// Você pode experimentar isso destacando o erro abaixo
// e optando por executar o _Quick Fix_ (Correção rápida).
```
image

File: packages/playground-examples/copy/pt/3-7/Fixits/Infer From Usage Changes.ts [hunk 4]
```diff
@@ -0,0 +1,35 @@
+//// { compiler: {  noImplicitAny: false }, order: 2 }
+
+// Na versão 3.7 do TypeScript há existência de 'dedução por uso'
+// deixando o codigo mais inteligênte, Você agora poderá usar
+// uma lita de importantes tipos conhecidos (string, number, array, Promise)
+// e deduzer se o uso do tipo combina com a API de objetos.
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/3-7/Fixits/Infer From Usage Changes.ts]:
```suggestion
// Na versão 3.7 do TypeScript a correção 'dedução por uso'
// ficou mais inteligente. Ela agora poderá usar uma lista conhecida
// de tipos importantes (string, number, array, Promise)
// e deduzir se o uso do tipo combina com a API destes objetos.
```

File: packages/playground-examples/copy/pt/3-7/Fixits/Infer From Usage Changes.ts [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+// Para o próximos pequenos exmplos, selecione o parâmetro das
+// funções, clique na lâmpada e selecione "Deduzier os tipos de Parâmetros..."
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/3-7/Fixits/Infer From Usage Changes.ts]:
```suggestion
// Para o próximos pequenos exemplos, selecione o parâmetro das
// funções, clique na lâmpada e selecione _Infer parameter types from usage_ (Deduzir os tipos de parâmetros pelo uso)
```
image

File: packages/playground-examples/copy/pt/3-7/Fixits/Infer From Usage Changes.ts [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+// Deduzir um array de número
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/3-7/Fixits/Infer From Usage Changes.ts]:
```suggestion
// Deduzir um array de números
```

File: packages/playground-examples/copy/pt/3-7/Fixits/Infer From Usage Changes.ts [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+function pushNumber(arr) {
+  arr.push(12);
+}
+
+// Deduzir uma promisse
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/3-7/Fixits/Infer From Usage Changes.ts]:
```suggestion
// Deduzir uma Promise
```

File: packages/playground-examples/copy/pt/3-7/Syntax and Messaging/Flattened Error Reporting.ts [hunk 8]
```diff
@@ -0,0 +1,65 @@
+//// { compiler: {  }, order: 3 }
+
+// As mensagens de erros no TypeScript pode ser as vezes um pouco verbosas...
+// Com a versão 3.7, nós tivemos colocado poucos casos a qual poderia ser
+// particularmente notórios.
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/3-7/Syntax and Messaging/Flattened Error Reporting.ts]:
```suggestion
// Às vezes, as mensagens de erros no TypeScript pode ser um pouco verbosas...
// Com a versão 3.7, nós tratamos alguns casos que poderiam ser
// particularmente notórios.
```

File: packages/playground-examples/copy/pt/3-7/Syntax and Messaging/Flattened Error Reporting.ts [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+
+let a = { b: { c: { d: { e: "string" } } } };
+let b = { b: { c: { d: { e: 12 } } } };
+
+a = b;
+
+// Antes, is levaria duas linhas de código por propriedade aninhada,
+// a qual rapidamente significava que as pessoas aprendiam a ler
+// mensagens de erro lendo a primeira e última linha de mensagem
+// de erro.
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/3-7/Syntax and Messaging/Flattened Error Reporting.ts]:
```suggestion
// Antes, isso levaria a duas linhas de código por propriedade aninhada.
// Isso rapidamente significou que as pessoas aprenderam a ler
// mensagens de erro lendo a primeira e última linha de mensagem
// de erro.
```

**Review by danilofuchs — CHANGES_REQUESTED** [review]
Great job!
We are translating some code in the examples as well, to greatly reduce the barrier of entry for Portuguese-only speakers.

You could translate the code to use `maiorValorPositivo` instead of `maxHighValue`, for instance. However, I would happily approve the PR with the code in English, after some small adjustments!

**Comment by 4SS1S** [comment]
Thank you @danilofuchs!

This Portuguese corrections is so much better! 😀

**Review by danilofuchs — APPROVED** [review]
LGTM

**Comment by orta** [comment]
LGTM

**Comment by orta** [comment]
Cool, #906 fixed the logic

**Pull request merged**